scripts/configure

```diff
@@ -243,6 +209,8 @@ fi
 # TODO: Update all the above code to use this simpler logic
 for template in "${NGINX_CONF_FILE}" "${BITCOIN_CONF_FILE}" "${LND_CONF_FILE}" "${TOR_CONF_FILE}" "${ELECTRS_CONF_FILE}" "${ENV_FILE}"; do
   # Umbrel
+  sed -i "s/<umbrel-version>/${UMBREL_VERSION}.local/g" "${template}"
+  sed -i "s/<hostname>/${DEVICE_HOSTNAME}/g" "${template}"
```

Comment:
This one shouldn't be with  `.local` after?

Reply:
Damn good spot, I added it on the wrong line